packages/flab/stringify.js

```diff
@@ -0,0 +1,9 @@
+console.log(
+	'module.exports = '+
+	require("fs").readFileSync("/dev/stdin", "utf-8")
```

Comment:
Ugh... this is only at build time.  Do we need to support windows even for _that_? 😩